### PR TITLE
Feature/change disruption rules to filters

### PR DESCRIPTION
**Pull request by jfeiglin**

### CODE REVIEW

File: mitm_service/tunneler/tunneler.py [hunk 1]
```diff
@@ -155,7 +172,7 @@ def forward_loop(self):
                 continue
 
             try:
-                disrupted_packet = self.disrupt_layers(data)
+                disrupted_packet = self.filter_layers(data)
             except DropPacketException:
                 # some filter in disrupt_layers raised a drop packet exception, so drop this packet
```
Comment (benjaminbaker-dev) [inline mitm_service/tunneler/tunneler.py]: fix comment disrupt - > layer :)

File: mitm_service/tunneler/tunneler.py [hunk 2]
```diff
@@ -50,25 +50,42 @@ def __init__(self, target_mac, gateway_mac, my_mac, target_ip, interface):
...
         """
-        Add a disruption rule at the given layer
+        Add a filter at the given protocol layer for a given priority
         :param layer: The layer number to insert this rule at (must be one of the options in L2Tunnel.PARSE_CAPABLE_LAYERS)
-        :param rule: A lambda that excepts this layer's header and payload, modifies them, and returns them modified
+        :param resolution_index: The index at which to insert the rule into the processing queue. Rules are resolved in
+                                 the order of their resolution index, so a rule with a low resolution index is resolved
+                                 before a rule with a high resolution index. If the resolution index is greater than the
+                                 number of filters in the processing queue, then it is inserted at the end of the queue
+        :param protocol_filter: A callable that excepts this layer's header and payload, modifies them, and returns them modified
```
Comment (benjaminbaker-dev) [inline mitm_service/tunneler/tunneler.py]: excepts -> accepts - this is because you did not do Devra

**Review by benjaminbaker-dev — CHANGES_REQUESTED** [review]
basically just spelling mistakes :)

File: mitm_service/tunneler/tunneler.py (hunk 2 above)
Comment (jfeiglin) [reply mitm_service/tunneler/tunneler.py]: no you idiot, its because i misspelled expects.

**Review by benjaminbaker-dev — APPROVED** [review]
:)

**Pull request merged**